subsys/shell/shell_log_backend.c

```diff
@@ -158,9 +173,19 @@ static void put(const struct log_backend *const backend, struct log_msg *msg)
 		}
 
 		break;
-
+	case SHELL_LOG_BACKEND_BLOCKING:
+		/* Even though log message is handled notify shell thread which
+		 * will print command buffer after the log message.
+		 */
+		if (IS_ENABLED(CONFIG_MULTITHREADING)) {
```

Comment:
What if multithreading is disabled? We erased command prompt, how we can see it now?